src/Views/FirmwareView.vala

```diff
@@ -291,7 +291,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         }
 
         if (!result) {
-            show_error_dialog (device, "Download of %s was not succesfull".printf (uri));
```

Comment:
"Download of %s" is a semantic error. "not successful" should be "unsuccessful", and "successful" is superfluous.
Nowhere else in elementary OS is "successful/ly" used.